io-pay-portal-fe/src/index.ts

```diff
@@ -32,12 +33,49 @@ document.addEventListener("DOMContentLoaded", () => {
   const organizationIdEl: HTMLInputElement | null =
     (document.getElementById("organizationId") as HTMLInputElement) || null;
 
+  // check if all fields are OK
+  function fieldsCheck() {
+    const checkedFields = document.querySelectorAll("input[data-checked]");
+
+    if (checkedFields?.length === inputFields?.length) {
+      verify?.removeAttribute("disabled");
+    } else {
+      verify?.setAttribute("disabled", "disabled");
+    }
+  }
+
+  // Add / remove validity to input elements
+  function toggleValid(el: any, isItValid: any) {
+    if (isItValid === true) {
+      el.parentNode.classList.remove("is-invalid");
+      el.parentNode.classList.add("is-valid");
+      el.classList.remove("is-invalid");
+      el.classList.add("is-valid");
+      el.setAttribute("data-checked", 1);
+    } else {
+      el.parentNode.classList.remove("is-valid");
+      el.parentNode.classList.add("is-invalid");
+      el.classList.remove("is-valid");
+      el.classList.add("is-invalid");
+      el.removeAttribute("data-checked");
+    }
+  }
+
   if (inputFields) {
     for (const inputEl of Array.from(inputFields)) {
       (inputEl as HTMLInputElement).addEventListener("focus", (evt: Event) => {
         const el = evt?.target;
         (el as HTMLInputElement).nextElementSibling?.classList.add("active");
       });
+      (inputEl as HTMLInputElement).addEventListener("keyup", (evt: Event) => {
+        const el = evt?.target as HTMLInputElement;
+        if (el.value.length > 0) {
```

Comment:
OK for now as test, but for the future I think it's better to check at least the _PaymentNotice_ ( _CodiceAvviso_ ) with something like [this](https://github.com/pagopa/io-pagopa-commons/blob/b1a4c0bc6da23ebb88f9ad2cb82a8d83fbe830c9/src/pagopa.ts#L88)